assets/css/custom/program.scss

```diff
@@ -1,114 +1,84 @@
-@import '../../../public/bootstrap/scss/functions';
-@import '../base/variables';
-@import '../base/mixins';
-@import '../../../public/bootstrap/scss/variables';
-@import '../../../public/bootstrap/scss/mixins';
-@import 'theme_settings';
+@import "../../../public/bootstrap/scss/functions";
+@import "../base/variables";
+@import "../base/mixins";
+@import "../../../public/bootstrap/scss/variables";
+@import "../../../public/bootstrap/scss/mixins";
+@import "theme_settings";
+@import "@material/circular-progress";
 
-button, .button {
-  color: white;
+button,
```

Comment:
something like should be consistent for all pages ~ move it to the base file .. buttons.scss?